src/book/03-guides/01-main-codebase/04-robocup-setup.mdx

```diff
@@ -0,0 +1,27 @@
+---
+section: Guides
+chapter: Main Codebase
```

Comment:
It's not quite this. More firmware. Maybe under hardware, since it's most close to Flashing a Robot? Not sure. It could even go in that page.